### ISSUE TITLE
User Group is offline

### ISSUE THREAD

**Issue by platypii** [jscad/OpenJSCAD.org#1119]
## Expected Behavior

User group link should work

## Actual Behavior

Link redirects to NodeBB homepage. Seems like the account expired or something.

## Steps to Reproduce the Problem

  1. Click "User Group" under the JSCAD help menu.

## Specifications

  - Version: 2.5.11
  - Platform: all
  - Environment: browser

**Comment by z3dev** [comment]
Yeah. I was hoping that it was just a temporary outage, as we didn't get any notice of extended down time. 😞

**Comment by z3dev** [comment]
The User Group is online again. 😃
